docusaurus/video/docusaurus/docs/api/basics/get_started.mdx

```diff
@@ -464,8 +511,13 @@ const address = `${resp.call.ingress.rtmp.address}/<callType>/<callID>/<apiKey>/
 <TabItem value="py" label="Python">
 
 ```py
-def hello_world():
-    print("Hello, world!")
+call_type = "your_call_type"
+call_id = "your_call_id"
+data = {}  # Add any data you need for the get_or_create_call method
+call = client.video.call(call_type, call_id)
+response = call.create(data=data)
+rtmp_address = response["call"]["ingress"]["rtmp"]["address"]
+address = f"{rtmp_address}/{call_type}/{call_id}/{VIDEO_API_KEY}/{client.create_token()}"
```

Comment:
> response["call"]["ingress"]["rtmp"]["address"] 

this should be the fully qualified URL with api key and call id info included (rtmps://stream.io/call/default/123?apikey)

the stream key should be created using client.create_token() (user_id needs to be there)

Reply:
use "livestream" call type ;)